### PR TITLE
Removing newline in description to comply with pypa/setuptools#1390

### PR DESCRIPTION
**Pull request by paulgavrikov**
See issue #49

### CODE REVIEW

File: tflite2onnx/__init__.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 NAME = 'tflite2onnx'
 VERSION = '0.3.1'
 __version__ = VERSION
-DESCRIPTION = """Convert TensorFlow Lite models to ONNX models.
-tflite2onnx v%s, https://github.com/jackwish/tflite2onnx""" % VERSION
+DESCRIPTION = 'Convert TensorFlow Lite models to ONNX models. ' \
```
Comment (jackwish) [inline tflite2onnx/__init__.py:10]:
```suggestion
DESCRIPTION =  "tflite2onnx v%s, https://github.com/jackwish/tflite2onnx" % VERSION
```

File: tflite2onnx/__init__.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
+              'tflite2onnx v%s, https://github.com/jackwish/tflite2onnx' % VERSION
```
Comment (jackwish) [inline tflite2onnx/__init__.py:11]:
```suggestion
```

**Review by jackwish — COMMENTED** [review]
Let's remove the _real description_ if we are going to make it one line.

**Comment by paulgavrikov** [comment]
I'd do it the other way round and kickout the url. [setuptools](https://packaging.python.org/guides/distributing-packages-using-setuptools/#description) states "Give a short and long description for your project.". But ultimately it is up to you of course!

**Comment by jackwish** [comment]
@paulgavrikov By _share the pip package you built_ I mean upload to the discussion issue #49. We don't include wheel in our code.

For the URL part, I was suggested to do that. As people can easily search for it, I agree that we can remove it to just left the name, version, and a short description.

File: tflite2onnx/__init__.py [hunk 3]
```diff
@@ -7,8 +7,7 @@
 NAME = 'tflite2onnx'
 VERSION = '0.3.1'
 __version__ = VERSION
-DESCRIPTION = """Convert TensorFlow Lite models to ONNX models.
-tflite2onnx v%s, https://github.com/jackwish/tflite2onnx""" % VERSION
+DESCRIPTION =  "tflite2onnx v%s, Convert TensorFlow Lite models to ONNX" % VERSION
```
Comment (jackwish) [inline tflite2onnx/__init__.py:10]:
```suggestion
DESCRIPTION = "tflite2onnx v%s, Convert TensorFlow Lite models to ONNX" % VERSION
```

**Review by jackwish — CHANGES_REQUESTED** [review]

**Review by jackwish — APPROVED** [review]

**Pull request merged**